app.css

```diff
@@ -0,0 +1,80 @@
+nav { font: 20px Copperplate; }
```

Comment:
What file is it, why he is not in your submission folder?
Remove all files not for review.